drivers/i2c/i2c_cc13xx_cc26xx.c

```diff
@@ -58,12 +58,18 @@ static int i2c_cc13xx_cc26xx_transmit(const struct device *dev,
 	const uint32_t base = get_dev_config(dev)->base;
 	struct i2c_cc13xx_cc26xx_data *data = get_dev_data(dev);
 
-	/* Sending address without data is not supported */
```

Comment:
I'm not 100% convinced that this statement is true, as I'm sure is the case for everyone else working with this part.

Let's ask the contributor with arguably the most experience with these parts.

@vanti - is there a way to perform an i2c scan using the simplelink I2C driverlib? If so, how do we do it?

Reply:
For reference, a good description of the goal is here:
https://hackaday.com/2015/06/25/embed-with-elliot-i2c-bus-scanning/

and TI's API is here:
https://software-dl.ti.com/simplelink/esd/simplelink_msp432e4_sdk/2.30.00.14/docs/driverlib/msp432e4/html/group__i2c__api.html

Reply:
@cfriedt  @stephanosio  Assume that we need to add more code to `i2c  scan` to support some SoC like CC13xx serial. any suggest?  Or maybe this is a half-baked idea, and we still need to fix the problem of sending only addresses.